AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/Location.java

```diff
@@ -41,6 +43,24 @@ public void setLocation(String location) {
         this.location = location;
     }
 
+    @Column(name = "longitude", length = 100, nullable = false)
+    public Double getLongitude() {
```

Comment:
Given that these are non-nullable, are we better off using the primitive type `double` rather than the wrapper `Double`?
Will raise this as a group discussion today

Reply:
Okay 😁